tests/sentry/api/endpoints/test_organization_dashboard_details.py

```diff
@@ -288,6 +297,40 @@ def test_add_widget(self):
         assert len(queries) == 1
         self.assert_serialized_widget_query(data["widgets"][5]["queries"][0], queries[0])
 
+    def test_add_widget_with_field_aliases(self):
+        data = {
+            "title": "First dashboard",
+            "widgets": [
+                {
+                    "title": "Errors per project",
+                    "displayType": "table",
+                    "interval": "5m",
+                    "queries": [
+                        {
+                            "name": "Errors",
+                            "fields": [],
```

Comment:
We should probably improve the validation to keep the length of fields and field aliases the same. Can we do a follow up PR for that?

Reply:
oh yes... sure thing

Reply:
I thought about it too but need to clarify some doubts first... will create a draft PR 😉